compiler/optimizer/CopyPropagation.cpp

```diff
@@ -1230,7 +1230,7 @@ void TR_CopyPropagation::replaceCopySymbolReferenceByOriginalIn(TR::SymbolRefere
                if (origNode->getNumChildren() == 2)
                   {
                   twoChildrenCase = true;
-                  TR_ASSERT(0, "Only in WCode we can add extra children\n");
+                  TR_ASSERT_FATAL(0, "Only in WCode we can add extra children\n");
```

Comment:
I'm not sure I follow how making this assert fatal resolves the problem since it does not operate on the nullness of a particular pointer (only "0").

For the record, I think this should be a fatal assert as you have it, but I'm not clear how this resolves the warning.

Reply:
Sorry, I should have explained. I'd have to recompile to be sure, but from what I recall, the actual compilation error was a little down from here, in one of the `if (twoChildrenCase)` blocks. (I think the `dumpOptDetails` call might have been the source). This is the only place where `twoChildrenCase` can be set to `true`, so the fatal assert makes those blocks unreachable.